configs/sites/hera/packages.yaml

```diff
@@ -22,11 +22,14 @@ packages:
       prefix: /apps/oneapi
   openmpi:
     externals:
-    - spec: openmpi@4.1.5~cuda~cxx~cxx_exceptions~java+lustre~memchecker+pmi~static~wrapper-rpath schedulers=slurm
-      prefix: /scratch1/NCEPDEV/jcsda/jedipara/spack-stack/openmpi-4.1.5
+    - spec: openmpi@4.1.6~atomics~cuda~cxx~cxx_exceptions~gpfs~internal-hwloc~internal-libevent~internal-pmix~java+legacylaunchers~lustre~memchecker~openshmem~orterunprefix+pmi+romio+rsh~singularity+static+vt+wrapper-rpath fabrics=ucx schedulers=slurm
       modules:
       - gnu/9.2.0
-      - openmpi/4.1.5
+      - openmpi/4.1.6_gnu9.2.0
```

Comment:
It is still with gnu/9.2.0?.. Could we push for using 12.2.0?.

Reply:
I assume you mean 13.2.0? I'd prefer not to make a last-minute change unless it's critically needed for an application. Could it wait until the 1.8.0 release?

Reply:
Ah okay I see, thanks. Would you be willing to update the Hera config to add the second gcc version?

Reply:
@RatkoVasic-NOAA was also building the gnu@12.2.0 compiler and openmpi@4.1.6 with it - if his testing is successful, we should use that version.